make datagrid row not clickable if rowClick returns false (#7411)
marmelab/react-admin: docs/Datagrid.md

| ------------------------------ | ------------------------------------------------------------- |
| `& .RaDatagrid-tableWrapper`   | Applied to the div that wraps table element                   |
| `& .RaDatagrid-table`          | Applied to the table element                                  |
| `& .RaDatagrid-thead`          | Applied to the table header                                   |
| `& .RaDatagrid-tbody`          | Applied to the table body                                     |
| `& .RaDatagrid-headerCell`     | Applied to each header cell                                   |
| `& .RaDatagrid-headerRow`      | Applied to each header row                                    |
| `& .RaDatagrid-row`            | Applied to each row                                           |
| `& .RaDatagrid-rowEven`        | Applied to each even row                                      |
| `& .RaDatagrid-rowOdd`         | Applied to each odd row                                       |
| `& .RaDatagrid-rowCell`        | Applied to each row cell                                      |
| `& .RaDatagrid-expandHeader`   | Applied to each expandable header cell                        |

> | `& .RaDatagrid-clickableRow`   | Applied to each row if `rowClick` prop evaluated to be truthy |

Reviewer: ```suggestion
| `& .RaDatagrid-clickableRow`   | Applied to each row if the `rowClick` prop is or returns truthy |
```
